src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -949,6 +949,48 @@ static std::string servicePathSubscriptionRegex(const std::string& servicePath,
 
 
 
+/* ****************************************************************************
```

Comment:
This seems to be a leftover. This block of commented code should be removed.

Reply:
fixed in c089631